docs/ja/docs/deployment/versions.md

```diff
@@ -0,0 +1,87 @@
+# FastAPIのバージョンについて
+
+**FastAPI** は既に多くのアプリケーションやシステムに本番環境で使われています。また、100%のテストカバレッジを維持しています。しかし、活発な開発が続いています。
+
+高頻度で新機能が追加され、定期的にバグが修正され、実装は継続的に改善されています。
+
+これが現在のバージョンがいまだに `0.x.x` な理由であり、それぞれのバージョンは破壊的な変更がなされる可能性があります。<a href="https://semver.org/" class="external-link" target="_blank">セマンティック バージョニング</a>の規則に則っています。
+
+**FastAPI** で本番用アプリケーションを今すぐに作成したら (すでに何度も経験しているかもしれませんが)、残りのコードが正しく動作するバージョンなのか確認しなければいけません。
+
+## `fastapi` のバージョンを固定
+
+最初にすべきことは、アプリケーションが正しく動作する **FastAPI** のバージョンを固定することです。
+
+例えば、バージョン `0.45.0` を使っているとしましょう。
+
+`requirements.txt` を使っているなら、以下の様にバージョンを指定できます:
+
+```txt
+fastapi==0.45.0
+```
+
+これは、バージョン `0.45.0` だけを使うことを意味します。
+
+または、以下の様に固定することもできます:
+
+```txt
+fastapi>=0.45.0,<0.46.0
+```
+
+これは `0.45.0` 以上、`0.46.0` 未満のバージョンを使うことを意味します。例えば、バージョン `0.45.2` は使用可能です。
+
+PoetryやPipenvなど、他のインストール管理ツールを使用している場合でも、それぞれパッケージのバージョンを指定する機能があります。
+
+## 利用可能なバージョン
+
+[Release Notes](release-notes.md){.internal-link target=_blank}で利用可能なバージョンが確認できます (最新版の確認などのため)。
+
+## バージョンについて
+
+Semantic Versioningの規約に従って、`1.0.0` 未満の全てのバージョンは破壊的な変更が加わる可能性があります。
```

Comment:
```suggestion
セマンティックバージョニングの規約に従って、`1.0.0` 未満の全てのバージョンは破壊的な変更が加わる可能性があります。
```
#L7 uses English term, so at least better to make them in the same representation.